releases/v1.1.md

```diff
@@ -0,0 +1,73 @@
+---
+title: What&#39;s New in v1.1
+toc: false
+summary: Additions and changes in CockroachDB version v1.1
+redirect_from: /v1.0.html
+---
+
+## DATE
+
+With the release of CockroachDB v1.1, we’ve included support for faster migrations from legacy databases for developers, simplified cluster management for operators, and improved performance in real-world environments. Read about the implications of these changes in [this blog post](https://www.cockroachlabs.com/blog/cockroachdb-1-0-release/), checkout a more granular [summary of new SQL, CLI, and Admin UI features](#summary) below, and then [upgrade to CockroachDB v1.1](../v1.1/upgrade-cockroach-version.html).
```

Comment:
"Legacy" is too harsh.

Don't forget to update the blog link

Reply:
Changed "legacy" to "other". Thanks for the reminder about the link. I don't have it yet, but I'll be sure to update this once I do.